AppCenter/AppCenter/Internals/Channel/MSChannelDelegate.h

```diff
@@ -87,6 +87,22 @@
               didSetEnabled:(BOOL)isEnabled
     andDeleteDataOnDisabled:(BOOL)deletedData;
 
+/**
+ * A callback that is called when pause has been invoked.
+ *
+ * @param channel The channel.
+ * @param identifyingObject The identifying object used to pause the channel.
```

Comment:
I like the API but I'd love to see some more explanation around this, e.g. what object the `identifyingObject` could be, and what "pausing" means. It's not a release blocker, just my opinion.